scvi/nn/_base_components.py

```diff
@@ -594,9 +595,9 @@ def forward(self, x: torch.Tensor, head_id: int, *cat_list: int):
 
         q_m = self.mean_encoder(q)
         q_v = torch.exp(self.var_encoder(q))
-        latent = reparameterize_gaussian(q_m, q_v)
-
-        return q_m, q_v, latent
+        dist = Normal(q_m, q_v.sqrt())
+        latent = dist.rsample()
+        return dist, latent
```

Comment:
I'm fine with this breaking change, but maybe want to be consistent?

Reply:
what do you mean exactly?

Reply:
i understand, on it